test/parallel/test-tls-transport-destroy-after-own-gc.js

```diff
@@ -0,0 +1,30 @@
+// Flags: --expose-gc
+'use strict';
+
+// Regression test for https://github.com/nodejs/node/issues/17475
+// Unfortunately, this tests only "works" reliably when checked with valgrind or
```

Comment:
Just for clarity: I assume "works" in this context means "doesn't raise memory leak errors"..? And properly running the test is just using [the `--valgrind` option](https://github.com/nodejs/node/blob/800ce94e5cdaa8dba7cf69b4c560a66a51ae0ec8/tools/test.py#L1601-L1605)?

Reply:
@jkrems Yeah – it complains about accessing released memory when run with valgrind. “Unfortunately”, most of the time that doesn’t lead to crashes…

Running the test file with `--valgrind` should make that reflected in the exit code, yes :)